lib/shared/severity.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
```

Comment:
Please add a file header comment describing the file.

Reply:
Added.